docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,30 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI (Unified Extensible Firmware Interface) secure boot when provisioning a host. This feature requires supported hardware and compatible OS image. The current hardwares that support enabling UEFI secure boot are `iLO driver`, `iRMC driver` and `Redfish driver`.
+
+Check also
+
+- [Ironic UEFI secure-boot](https://docs.openstack.org/ironic/latest/admin/security.html#uefi-secure-boot-mode)
+- [Wikipeda UEFI secure-boot](https://en.wikipedia.org/wiki/UEFI#SECURE-BOOT)
+
+## Advantages/Benefits
+
+We need the automatic secure boot when provisioning a host with high security requirements. Based on checksum and signature, the secure boot protects the host from loading malicious code in the boot process before loading the provisioned operating system.
+
+## Usage
+
+To enable Automatic secure boot check that the hardware is supported then specify the value `UEFISecureBoot` for `bootmode` in the bmh custom resource. Check the example:
```

Comment:
typo: bootMode

Reply:
I'd also spell out BareMetalHost since the "bmh" shortening is not used in this document

Reply:
thanks done